miasm2/expression/expression.py

```diff
@@ -511,6 +522,9 @@ def __new__(cls, name, size=32):
     def __str__(self):
         return str(self.__name)
 
+    def str_sub_expr(self):
```

Comment:
This method can be declared in Expr, and then there is no need to repeat it if there is no change